Xabe.FFmpeg.Test/ConversionTests.cs

```diff
@@ -120,6 +120,28 @@ public async Task SetOutputPixelFormatTest()
             Assert.Equal("yuv420p", resultFile.VideoStreams.First().PixelFormat);
         }
 
+        [Fact]
```

Comment:
Please change it to theory and test every value from HashFormat

Reply:
Done, I don't know if there is a way to improve this so we don't need the switch statement?

Reply:
Just pass length as a second parameter to function :)
[InlineData("MD5", 37L)]
public async Task SetHashFormatTest(string hashFormat, long expectedLength)

Reply:
Done